assets/src/dashboard/constants/pageStructure.js

```diff
@@ -17,9 +17,11 @@
 /**
  * Internal dependencies
  */
-import { PAGE_HEIGHT, PAGE_WIDTH } from '../../edit-story/constants';
+import { FULLBLEED_RATIO } from '../../edit-story/constants';
+
 export const WPBODY_ID = 'wpbody';
-export const PAGE_RATIO = PAGE_WIDTH / PAGE_HEIGHT;
+export const PAGE_RATIO = FULLBLEED_RATIO;
+export const TWO_THIRDS_RATIO = 2 / 3;
```

Comment:
I think we should follow the naming conventions of the `edit-story` code base.  Their `constants` file already has a definition for `PAGE_RATIO` (2/3) and `FULLBLEED_RATIO` (9/16) and I think we should just export them from this page (so our code would still import from `pageStructure`).  This way we're sharing the same ratios and names across codebases.